packages/automerge-repo/src/DocHandle.ts

```diff
@@ -419,37 +383,40 @@ export class DocHandle<T> //
     })
   }
 
+  /** Marks this document as unavailable. */
```

Comment:
this should probably be hidden and/or more clearly named since it's not meant to be user-facing

Reply:
(as the rest of the methods in this section already are!)

Reply:
Ah, this is only used by one test. Comment updated.